### PR TITLE
35137 Print view update

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Description
This PR updates the appointment list printout

## Original issue(s)
department-of-veterans-affairs/va.gov-team#35137

## Testing done
N/A

## Screenshots
Screen Shot 2022-01-19 at 1 12 09 PM
---
Screen Shot 2022-01-19 at 1 13 28 PM
---
Screen Shot 2022-01-19 at 1 20 00 PM
---
Screen Shot 2022-01-19 at 1 20 42 PM

## Acceptance criteria
- [ ] All test should pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ function getDropdownValueFromLocation(pathname) {
     return {
       dropdownValue: DROPDOWN_VALUES.requested,
       subPageTitle: 'Requested',
-      subHeading: 'Requested',
+      subHeading: 'Requested appointments',
```
Comment (vbahinwillit) [inline src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx:38]: Change per conversation with Peter to be consistent with the other pages.

File: src/applications/vaos/tests/appointment-list/components/AppointmentsPageV2/index.unit.spec.jsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ describe('VAOS <AppointmentsPageV2>', () => {
     expect(
       screen.getByRole('heading', {
         level: 2,
-        name: 'Requested',
+        name: 'Requested appointments',
```
Comment (vbahinwillit) [inline src/applications/vaos/tests/appointment-list/components/AppointmentsPageV2/index.unit.spec.jsx:70]: Change per conversation with Peter to be consistent with the other pages.

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx [hunk 3]
```diff
@@ -75,6 +75,28 @@ export default function AppointmentsPageV2() {
...
+      window.addEventListener('beforeprint', handleBeforePrint);
+      window.addEventListener('afterprint', handleAfterPrint);
+      return () => {
+        window.removeEventListener('beforeprint', handleBeforePrint);
+        window.removeEventListener('afterprint', handleAfterPrint);
+      };
+    },
+    [documentTitle, subPageTitle],
+  );
+
```
Comment (jenniemc) [inline src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx:99]: Great solution to always print "Your appointments | VA Online Scheduling | Veterans Affairs" as the header in the print version of upcoming/requested/past/cancel appointment list.

**Review by jenniemc — APPROVED** [review]
LGTM!

**Review by ptouchton — APPROVED** [review]

**Pull request merged**